package.json

```diff
@@ -11,7 +11,7 @@
     "unit": "tap --test-regex='(\\/|^tests\\/unit\\/.*\\.test\\.js)$' --no-coverage",
     "versioned": "npm run versioned:npm6",
     "versioned:folder": "versioned-tests --minor -i 2",
-    "versioned:npm6": "versioned-tests --minor -i 2 'tests/versioned/*'",
+    "versioned:npm6": "versioned-tests --minor -i 2 'tests/versioned/**/!(apollo-server-koa)' && versioned-tests --minor --all -i 2 'tests/versioned/apollo-server-koa'",
```

Comment:
this is explained in PR description but now on npm6 it will run all versioned tests but koa in 1 run and then just koa in another with the `--all` flag